[2022-09-21T22:19:58.392+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag_example.create_pet_table manual__2022-09-21T22:19:53.096618+00:00 [queued]>
[2022-09-21T22:19:58.407+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag_example.create_pet_table manual__2022-09-21T22:19:53.096618+00:00 [queued]>
[2022-09-21T22:19:58.408+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-21T22:19:58.412+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2022-09-21T22:19:58.413+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-21T22:19:58.430+0000] {taskinstance.py:1389} INFO - Executing <Task(PostgresOperator): create_pet_table> on 2022-09-21 22:19:53.096618+00:00
[2022-09-21T22:19:58.439+0000] {standard_task_runner.py:52} INFO - Started process 62988 to run task
[2022-09-21T22:19:58.443+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag_example', 'create_pet_table', 'manual__2022-09-21T22:19:53.096618+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_demo.py', '--cfg-path', '/tmp/tmpe853q8q2', '--error-file', '/tmp/tmphfwuaf0f']
[2022-09-21T22:19:58.454+0000] {standard_task_runner.py:80} INFO - Job 13: Subtask create_pet_table
[2022-09-21T22:20:09.049+0000] {task_command.py:371} INFO - Running <TaskInstance: postgres_operator_dag_example.create_pet_table manual__2022-09-21T22:19:53.096618+00:00 [running]> on host 49fa11dd9034
[2022-09-21T22:20:09.434+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=jsanchez
AIRFLOW_CTX_DAG_ID=postgres_operator_dag_example
AIRFLOW_CTX_TASK_ID=create_pet_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-21T22:19:53.096618+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-21T22:19:53.096618+00:00
[2022-09-21T22:20:09.520+0000] {base.py:68} INFO - Using connection ID 'qa_ai_external_db' for task execution.
[2022-09-21T22:20:10.347+0000] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS ***_demo.pet (
                pet_id SERIAL PRIMARY KEY,
                name VARCHAR NOT NULL,
                pet_type VARCHAR NOT NULL,
                birth_date DATE NOT NULL,
                OWNER VARCHAR NOT NULL), parameters: None
[2022-09-21T22:20:10.916+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag_example, task_id=create_pet_table, execution_date=20220921T221953, start_date=20220921T221958, end_date=20220921T222010
[2022-09-21T22:20:11.287+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-21T22:20:11.487+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
